### PR TITLE
Example for pdf generation

### PR DESCRIPTION
**Pull request by kitsane**
Only as example for setup and how to generate a pdf, not for merge

### CODE REVIEW

File: test/models/assignment_test.rb [hunk 1]
```diff
@@ -0,0 +1,7 @@
+require 'test_helper'
+
+class AssignmentTest < ActiveSupport::TestCase
+  # test "the truth" do
+  #   assert true
+  # end
+end
```
Comment (alesrosina) [inline test/models/assignment_test.rb]: You can remove this file, since it is empty.

File: app/controllers/clients_controller.rb [hunk 2]
```diff
@@ -44,6 +44,11 @@ def destroy
     redirect_to clients_url, make_notice
   end
 
+  def without_assignments
```
Comment (alesrosina) [inline app/controllers/clients_controller.rb]: Maybe rename to `need_accompanying`, so you would have nicer routes file (see also comment for `routes.rb`)

File: app/controllers/assignments_controller.rb [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+    client.state = Client::REGISTERED
+    client.save
+    if Volunteer.without_clients.include?(volunteer)
+      volunteer.state = Volunteer::ACCEPTED
+      volunteer.save
+    end
+    redirect_to assignments_url, make_notice
+  end
+
+  def find_volunteer
```
Comment (alesrosina) [inline app/controllers/assignments_controller.rb]: Maybe move to `ClientController`, so you would have nicer routes file (see also comment for routes.rb).

File: config/routes.rb [hunk 4]
```diff
@@ -1,4 +1,8 @@
 Rails.application.routes.draw do
+  get '/clients/need_accompanying', to: 'clients#without_assignments'
+  get '/clients/find_volunteer', to: 'assignments#find_volunteer'
```
Comment (chrysalag) [inline config/routes.rb]: I have them like this because it was in the requirements to have such urls.

**Pull request merged**